.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:

```

Comment:
add an `e.g.` to make clear that they should tweak these steps and not try to shoehorn their workflow into the ellipses